### PR TITLE
Add devise sign in

### PR DESCRIPTION
**Pull request by kian1213**
Hi sir, please review my pull request. 

the complete feature will be added in next PR. I already have 15 files here. Thanks po.

### CODE REVIEW

**Comment by codegracie** [comment]
Okay

File: app/assets/stylesheets/admin/styles_admin.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
+.admin-actions {
+  text-align:center
+}
+
+.logout-button {
+  text-align: right;
+}
+
+.wrapper {
+    display: inline-block;
```
Comment (dieunb) [inline app/assets/stylesheets/admin/styles_admin.scss]: should follow indent = 2 spaces with css

File: app/assets/stylesheets/admin/simple-sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
+#wrapper {
+    padding-left: 0;
```
Comment (dieunb) [inline app/assets/stylesheets/admin/simple-sidebar.scss]: should follow indent = 2 spaces with css
 Add a line note

File: app/controllers/admin_controller.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class AdminController < ApplicationController
+  layout "admin"
+  before_action :authenticate_user!, :ensure_user_rights
+
+  private
+  def ensure_user_rights
```
Comment (dieunb) [inline app/controllers/admin_controller.rb]: on the next time, you can implement `cancancan` to handle it.

**Pull request merged**